### ISSUE TITLE
Section Changes on Results Filters

### ISSUE THREAD

**Issue by annatuma** [creativecommons/cccatalog-frontend#910]
We want to make a few small changes to the Filters on the Results page.

[This is the latest design iteration](https://www.figma.com/file/KEXxItd8EloJGbXswuUDqq/Mockup-S3-License-Language-Changes?node-id=199%3A353).

1) We want to remove the Funnel icon and replace it with "Show/Hide" filters language, per the mockup, to make this action more clear. (This was completed in https://github.com/creativecommons/cccatalog-frontend/pull/867)

2) We're changing the name of "Collections" to "Sources".

3) We're removing the "Settings" section, where the Mature Content filter flag is, and instead placing that at the top right of search results. Addressed here: https://github.com/creativecommons/cccatalog-frontend/issues/1033

**Comment by abhisheknaiidu** [comment]
Can I work on this @annatuma ?

**Comment by krysal** [comment]
This seems to be related with my PR #867. @annatuma Could you please share the link with the updated mockup?

**Comment by DantrazTrev** [comment]
@annatuma I would like another change for the filter section , the filter  Image size is really a vague and not properly labelled filter . Considering form a user side the filter could be interpreted anything from Image file size to Image dimensions and further the boundary between the given three options remains vague to the end user . A good idea would be to include the size on the filter section or an alternative could be it included in the search guide which wouldn't be much of the help to not the savvy users.

**Comment by Fahme** [comment]
@annatuma If no one is picking this up i would love to help ?

**Comment by annatuma** [comment]
Anyone who would like to tackle this is welcome to open a PR and get started. Items 2 and 3 in the ticket still need to be done.

@krysal sorry about that, I've updated the ticket to include the mockup link. The first change listed here ended up being done in your PR #867 and so I've indicated that in the ticket.

@DantrazTrev that's good feedback regarding the Image Size filter and lack of clarity. Please kick off a conversation in the #cc-search channel in Slack. I don't have handy what small/medium/large correlates to in terms of dimensions, but we can discuss how to communicate those dimension ranges usefully.

**Comment by Fahme** [comment]
Hey @annatuma 

Thanks, i think i can pick up this task 

> We're changing the name of "Collections" to "Sources".

However i just need to clarify on few points before sending a PR, because the issue only mentioned to change "Collections" to "Sources".

- Should the components names for the collection pages be changed as well to sources to make it more clear in the codebase ?
- I would assume routes as well on the FE application should be changed from ``/collections`` to ``/sources``  ?
- Test should be renamed as well ?

basically anything related to collections like Props, functions names should be addressed to suit the new naming of the page, right ?

**Comment by Tanuj22** [comment]
@annatuma I have fixed point 3 in #918 . Please have a look.

**Comment by annatuma** [comment]
@Fahme I defer to @brenoferreira for what is appropriate to do regarding component names, routes, etc.

**Comment by annatuma** [comment]
All of the work detailed in this ticket was completed (in various PRs).
